eras/shelley/impl/src/Cardano/Ledger/Shelley/Rules/PoolReap.hs

```diff
@@ -165,7 +165,7 @@ poolReapTransition = do
       us {utxosDeposited = utxosDeposited us <-> (unclaimed <+> refunded)}
       a {asTreasury = asTreasury a <+> unclaimed}
       ( let u0 = dsUnified ds
-            u1 = (Rewards u0 UM.∪+ refunds)
+            u1 = (Rewards u0 UM.∪+ (Map.map compactCoinOrError refunds))
```

Comment:
All of these parenthesis are redundant (same for the line below, but I can't make a suggestions, since it hasn't changed.
```suggestion
            u1 = Rewards u0 UM.∪+ Map.map compactCoinOrError refunds
```